index.html

```diff
@@ -34,12 +34,236 @@ <h3>I am a software developer</h3>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+                <section class="card-works">
+                    <section class="mask">
+                        <section class="img-placeholder">
+                            <section class="div">
+                                <section class="right-works">
+                                    <p class="pro">Profesional Art Printing Data</p>
+                                    <p class="sub1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    <ul class="tags">
+                                        <li class="tags1"><h6>html</h6></li>                                            
+                                        <li class="tags2"><h6>bootstrap</h6></li>
+                                        <li class="tags3"><h6>Ruby</h6></li>
+                                    </ul>
+                                </section>
+                            </section>
+                        </section>
+                        <!-- <section class="mask-sub"></section> -->
+                    </section>
+                    <section class="action">
+                        <section class="but">
+                            <p class="see1">See Project</p>
+                        </section>
+                    </section>
+                </section>
+            </section>
+            <section class="about">
+                <p class="primary">About Me</p>
+                <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <section class="img-about"><img src="images/illustration-about-me-mobile.png" alt=""></section>
+                <section class="button">
+                    <p class="get">Get My Resume</p>
+                </section>
+                <section class="divider">
+                    <div class="divi"></div>
+                </section>
+                <section class="skill">
```

Comment:
- [ ] In the skills section, as the last code reviewer said: Please 🙏🏽  align the list of technologies to look like the Figma template. You are expected to stick to the design template as much as possible.

- **Your design**
![image](https://user-images.githubusercontent.com/58642949/192076580-55fd170e-33c7-4552-bd41-098b8a8f3804.png)

- **Figma design**
![image](https://user-images.githubusercontent.com/58642949/192076598-397f4a99-e2a1-45ec-9265-7ff80c82f3d1.png)